vtex-apps/product-specifications
# Feature/specification group filter

#### What does this PR do? \*

Add condition from specification name to show specifications.

#### How to test it? \*

1 - Enter the following WS: [Workspace](https://sdtio713--diageo.myvtex.com/Delicia-De-Natal-Baileys/p)

2 - Scroll to the bottom of the page and see that before the footer there is duplicate content where we have a shelf [Image 1](https://flic.kr/p/2kJJXc4)

3 - On this shelf, only two specifications and their respective values ​​are being shown [Image 2](https://flic.kr/p/2kJJXbn)

4 - If, in the code, you remove the conditionBySpecName prop or leave it empty, all specifications will be shown [Image 3](https://flic.kr/p/2kJK32e)
